Makefile.am

```diff
@@ -123,6 +125,9 @@ noinst_LTLIBRARIES = \
 	libmetadata.la \
 	libmount.la \
 	liboconfig.la
+if !BUILD_WIN32
```

Comment:
Could you please move this up so the two options are closer together:

```
if BUILD_WIN32
lib_LTLIBRARIES …
else
noinst_LTLIBRARIES …
endif
```

Reply:
Done.